src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -4484,3 +4484,54 @@ def test_aks_update_to_msi_cluster_with_addons(self, resource_group, resource_gr
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @live_only()
```

Comment:
Will this test case fail in recording mode? If not, we should remove ```@live_only()```.

Reply:
Makes sense, updated.

Reply:
I've added it back because test case fail in recording mode https://github.com/Azure/azure-cli/runs/2930744551 . Seems it is due to `_get_user_assigned_identity` returns a `None` object. Not very clear about the reason, checked recording file, there does exist mock for GET userAssignedIdentities, I don't know how to solve the error so I just added live_only back.